daemon/daemon.go

```diff
@@ -41,6 +42,7 @@ type Daemon struct {
 	listener     net.Listener
 	tomb         tomb.Tomb
 	router       *mux.Router
+	capRepo      *capabilities.CapabilityRepository
```

Comment:
@zyga checked with me before putting it here; I'll repeat here what I told him: without knowing how it's going to get used it's hard to give an answer with certainty, but you can't go wrong putting it in the daemon, as there'll only be one instance of it and every command has access to it.